extensions/cornerstone/src/components/ViewportDataOverlaySettingMenu/utils.ts

```diff
@@ -0,0 +1,141 @@
+import { utilities as csUtils } from '@cornerstonejs/core';
+
+export const DEFAULT_COLORMAP = 'hsv';
+export const DEFAULT_OPACITY = 0.5;
+export const DEFAULT_OPACITY_PERCENT = DEFAULT_OPACITY * 100;
+export const DERIVED_OVERLAY_MODALITIES = ['SEG', 'RTSTRUCT'];
+
+/**
+ * Get modality-specific color and opacity settings from the customization service
+ */
+export function getModalitySettings(customizationService, modality) {
```

Comment:
Can you call this something descriptive enough to know what sort of thing it returns?
Maybe getModalityOverlayColorMap ?